inventory_management/inventory_management/report/stock_balance/stock_balance.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2024, Viny Selopal and contributors
+// For license information, please see license.txt
+
+frappe.query_reports["Stock Ledger"] = {
```

Comment:
```suggestion
frappe.query_reports["Stock Balance"] = {
```
Keep most used filters in the beginning - eg: From Date and To Date can be in the beginning and defaulted to the current month maybe so that we don't end up reading the entire table